src/shared/nsresource.c

```diff
@@ -15,8 +15,12 @@
 #include "process-util.h"
 #include "string-util.h"
 
+/* Maximum namespace name length */
+#define NAMESPACE_NAME_MAX 16U
+
 static int make_pid_name(char **ret) {
         char comm[TASK_COMM_LEN];
```

Comment:
NAMESPACE_NAME_MAX here?

Reply:
nah, this is space for PR_GET_NAME, we really should use TASK_COMM_LEN for that.